proto/rpc.proto

```diff
@@ -1,6 +1,6 @@
 syntax = "proto3";
 
-
+import "google/api/annotations.proto";
```

Comment:
For historical purposes, we actually remove this line w/ `sed` in the build.sh. but since we changed the name of this file (from lnd-rpc to rpc), this change is now added.

We should remove the associated change in build.sh here: https://github.com/sparkswap/lnd-engine/blob/master/scripts/build.sh#L42-L44